index.js

```diff
@@ -17,13 +18,23 @@ log.type('error', {
 	prefix: figures.cross
 });
 
-function test(err, title) {
+function test(err, title, duration) {
 	if (err) {
 		log.error(title, chalk.red(err.message));
 		return;
 	}
 
-	log.success(title);
+	// format a time spent in the test
+	var timeSpent = '';
+
+	// display duration only over a threshold
+	var threshold = 100;
+
+	if (duration > threshold) {
+		timeSpent = chalk.dim('(' + prettyMs(duration) + ')');
```

Comment:
Just remembered that `dim` doesn't work everywhere. Can you make it `chalk.gray.dim`?